main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -188,6 +189,49 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
     var level = DialogSystem.showDialog(frame);
   };
 
+  var doAddIdcolumn = function() {
+    var frame = $(
+      DOM.loadHTML("core", "scripts/views/data-table/add-q-column-dialog.html"));
+
+    var elmts = DOM.bind(frame);
+    elmts.dialogHeader.text($.i18n('core-views/add-id-col')+" " + column.name);
```

Comment:
Localizing text in this way makes it impossible for translators to reorder the column name to a different position in the message. I am really not blaming you since there are loads of places where we do that in the UI at the moment - I'll open an issue to clean these up.

Because we are introducing a new message here, let's try to get this right. We use the Wikimedia jQuery i18n plugin, which lets you define templates like this:
https://github.com/wikimedia/jquery.i18n#jqueryi18n-plugin

So you should be able to pass `column.name` as an extra argument to the function and insert it with `$1` in the message text.

Reply:
Got it!

Reply:
Not sure if I was supposed to click Resolve on my own xD Sent a final pull request!